test/bigtest/tests/package-content-test.js

```diff
@@ -0,0 +1,126 @@
+  accessEnd: () => faker.date.recent().toISOString()
+};
+
+describe('Package Content Filters', () => {
+  setupApplication();
+  const packagecontent = new PackageContentInteractor();
+  const basket = new BasketInteractor();
+  const agreementform = new AgreementFormInteractor();
+  const agreementview = new AgreementViewInteractor();
+
+  let eresource;
+
+  beforeEach(async function () {
+    eresource = this.server.create('eresource', { name: PKG.name, class: 'org.olf.kb.Pkg', pcis: [droppedResource, currentResource, futureResource] });
+  });
+
+  describe('packagecontent tests', () => {
+    beforeEach(async function () {
+      this.visit(`/erm/eresources/${eresource.id}`);
+    });
+
+    describe('visiting the eresource pane', () => {
+      it('should render expected package', () => {
+        expect(packagecontent.headline).to.equal(PKG.name);
+      });
+
+      it('should render the expected current eresource', () => {
+        expect(packagecontent.eresourceName).to.equal(currentResource.pti.titleInstance.name);
+      });
+
+      describe('clicking the future eresource tab', () => {
+        beforeEach(async function () {
+          await packagecontent.clickFuture();
+        });
+
+        it('should render the expected future resource', () => {
+          expect(packagecontent.eresourceName).to.equal(futureResource.pti.titleInstance.name);
+        });
+      });
+
+      describe('clicking the dropped eresource tab', () => {
+        beforeEach(async function () {
+          await packagecontent.clickDropped();
+        });
+
+        it('should render the expected dropped resource', () => {
+          expect(packagecontent.eresourceName).to.equal(droppedResource.pti.titleInstance.name);
+        });
+      });
+
+      describe('Add eresource to basket and create agreement', () => {
+        beforeEach(async function () {
+          await basket.addToBasketButtion();
+          await basket.clickOpenBasket();
+          await basket.clickCreateNewAgreement();
+          await agreementform.fillName('testAgreement');
+          await agreementform.selectStatus('Draft');
+          await agreementform.processDate('2020-04-01');
+          await agreementform.createAgreement();
+          await agreementview.whenLoaded();
+        });
+
+        describe('should open agreement view pane', () => {
+          describe('clicking current eresource', () => {
+            beforeEach(async function () {
+              await agreementview.clickLinesAccordion();
+            });
+
+            it('should render the expected current eresource', () => {
+              expect(packagecontent.eresourceName).to.equal(currentResource.pti.titleInstance.name);
+            });
+          });
+
+          describe('clicking future eresource', () => {
+            beforeEach(async function () {
+              await agreementview.clickLinesAccordion();
+              await packagecontent.clickFuture();
```

Comment:
In an ideal world, this would probably be something like `await agreementView.linesSection.coveredEresourcesList.clickFuture()` but it's up to you. 

Because of course, we're _not_ on the package content view, and it's just good fortune that the same selector works here. But we can't assume that the package contents view and the covered eresources view will stay the same. In fact, they'll probably be different very soon, when we add the ability to perform searches on the list in the Package Contents section.

Reply:
Yes. Added a `CoveredEresourcesListInteractor` within `AgreementViewInteractor` that has the clicks on the filters.